pages/index.vue

```diff
@@ -93,9 +95,15 @@
                 "dataTable"
             ]),
 
+            dataTableSelected() {
+
```

Comment:
I think this guy may benefit from a small comment explaining why it is here. I.e.: checking if dataTable is already there. For future us mainly